CHANGELOG.md

```diff
@@ -4,6 +4,13 @@ All notable changes to this project will be documented in this file.  The format
 Changelog](https://keepachangelog.com/en/1.0.0/).
 
 ## Unreleased
+## 0.3.1 11-21-2020
+### Changed
+- Simplified standalone collector code. (#83, @stati)
+- Improved intermediate process. (#81, @zyiou)
+- Exposed collecting process and aggregation process as public struct. (#84, @zyiou)
+### Fixed
+- Modified go.mod for inconsistency with Antrea. (#82, @zyiou)
```

Comment:
Rephrasing it a bit:
Modified versions of some packages in go.mod to keep it consistent with Antrea, the main user of go-ipfix library